frontend/App.js

```diff
@@ -34,6 +36,13 @@ function MainTabs({ navigation }) {
 }
 
 function App() {
+	useEffect(() => {
+        const testAPI = async () => {
+            await UtilWrite();
+        };
+        testAPI();
+    }, []);
+
```

Comment:
If you're sure that this works, lets clean it up so we don't leave it running in our `App` component. Might break people's code when they pull in master if they don't have AWS configured.